src/main/java/mate/controller/GetAllCarsController.java

```diff
@@ -0,0 +1,24 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+public class GetAllCarsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Car> carList = carService.getAll();
+        req.setAttribute("cars", carList);
+        req.getRequestDispatcher("/WEB-INF/views/cars/allCars.jsp").forward(req, resp);

```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/cars/all.jsp").forward(req, resp);
```
The same for other GetAll controllers